LogControl/Domain/Entity/Log.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System.ComponentModel.DataAnnotations;
+
+namespace LogControl.Domain.Entity
+{
+    public class Log
+    {
+        public Guid Id { get; set; } = Guid.NewGuid();
+        public string Level { get; set; }
+        public string MicroserviceName { get; set; }
+        public string Message { get; set; }
+        public string? Exception { get; set; }
+        public string Environment { get; set; }
+        public DateTime Time { get; set; } = DateTime.Now;
```

Comment:
Better to use `DateTime.UtcNow`  on the BE because otherwise you field will depend on the env it's deployed (but it can be also fine for some specific business cases, but the general practice is to use UTC I believe)